docs/docs/developer-guides/cosmwasm/walkthrough.md

```diff
@@ -95,7 +65,7 @@ make use of SS58 representations in the contracts as well, but for now, this is
 
 So the command will be:
 ```sh
-ccw substrate -c ws://127.0.0.1:9988 -n alice \
+ccw substrate -c wss://picasso-rococo-unrpc-lb.composablenodes.tech/ -n alice \
```

Comment:
i guess -n alice should be replaced with some private key or seed phrase for example

Reply:
I haven't tested yet but yeah I assumed so. I will update the docs once tested.